manifest_staging/charts/secrets-store-sync-controller/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: secrets-store-sync-controller
 version: 0.0.1
 appVersion: v0.0.1
-kubeVersion: ">=1.27.0"
+kubeVersion: ">=1.27.0-0"
```

Comment:
Odd the `-0` is what fixes this.  🤔

Reply:
For historical context, this is to match what is in the CSI Driver Repo.
Still not sure "why". 😄

Reply:
@stlaz comment (below in review) seconds this curiosity.